pkg/blockdevice/v1alpha1/blockdevice.go

```diff
@@ -29,20 +31,28 @@ import (
 //TODO: Update the file with latest pattern
 const (
 	// StorageNodePredicateKey is the key for StorageNodePredicate function.
-	FilterInactive       = "filterInactive"
-	FilterNonInactive    = "filterNonInactive"
-	FilterClaimedDevices = "filterClaimedDevices"
-	InActiveStatus       = "Inactive"
+	FilterInactive         = "filterInactive"
+	FilterNonInactive      = "filterNonInactive"
+	FilterNonPartitions    = "filterNonPartitions"
+	FilterSparseDevices    = "filterSparseDevices"
+	FilterNonSparseDevices = "filterNonSparseDevices"
+	InActiveStatus         = "Inactive"
 )
 
-// DefaultDiskCount is a map containing the default disk count of various raid types.
+// DefaultDiskCount is a map containing the default block device count of various raid types.
 var DefaultDiskCount = map[string]int{
 	string(apis.PoolTypeMirroredCPV): int(apis.MirroredBlockDeviceCountCPV),
 	string(apis.PoolTypeStripedCPV):  int(apis.StripedBlockDeviceCountCPV),
 	string(apis.PoolTypeRaidzCPV):    int(apis.RaidzBlockDeviceCountCPV),
 	string(apis.PoolTypeRaidz2CPV):   int(apis.Raidz2BlockDeviceCountCPV),
 }
 
+// SupportedDiskType is a map containing the valid disk type
+var SupportedDiskType = map[string]bool{
```

Comment:
why do we want to put this there? this is nothing to do with blockdevice.
Please move to maya related files

Reply:
moved to `pkg/storagepoolclaim/v1alpha1`